VTEX - Marketplace APIs - Suggestions.json

```diff
@@ -1635,6 +1714,70 @@
+                            },
+                            "Reject": {
+                                "type": "integer",
+                                "description": "Bounce score set by marketplace."
+                            }
+                        }
+                    },
+                    "Matchers": {
+                        "description": "Array of objects that presents a list of matchers configured in the marketplace. If the marketplace has more than one matcher configured in the account, an array will be displayed for each matcher.",
+                        "type": "array",
+                        "items": {
+                            "type": "object",
+                            "properties": {
+                                "MatcherId": {
+                                    "type": "string",
+                                    "description": "The Matcher Id."
+                                },
+                                "hook-base-address": {
+                                    "type": "string",
+                                    "description": "The base address of the Matcher hook."
+                                },
+                                "IsActive": {
+                                    "type": "boolean",
+                                    "description": "Indicates if Matcher is active or not."
+                                }
+                            }
+                        }
+                    },
+                    "Rules": {
+                        "description": "Arrays object that contains the rules defined for product approval.",
+                        "type": "object",
+                        "properties": {
+                            "Item": {
+                                "type": "array",
+                                "description": "A list of rules for items."
+                            },
+                            "Product": {
+                                "type": "array",
+                                "description": "A list of rules for products."
+                            }
+                        }
+                    },
+                    "SpecificationsMapping": {
+                        "type": "array",
+                        "description": "List of specifications mapping."
+                    },
+                    "MatchFlux": {
+                        "type": "string",
+                        "description": "This field determines the type of approval configuration applied to SKUs received from a seller. \n\nThe possible values include: \n\n-`default`, where the Matcher reviews the SKU, and approves it based on its score. \n\n-`manual`, for manual approvals through the Received SKU UI, or Match API. \n\n-`autoApprove`, for every SKU received from a given seller to be approved automatically, regardless of their Matcher Score."
```

Comment:
Como isso é diferente das rules aplicadas à items?

Reply:
Mudaria a vírgula pra dois pontos. Ex:
- `default`: Where the Matcher reviews the SKU, and approves it based on its score.

Reply:
Pensa que nossas regras editoriais do guia de estilo também se aplicam à dev docs